content/departments/cloud/index.md

```diff
@@ -78,9 +78,10 @@ The Cloud team roadmap in available [here](https://docs.google.com/spreadsheets/
 
 ## How to contact the team and ask for help
 
-- For emergencies and incidents, alert the team using Slack command `/genie alert [message] for cloud`.
-- For internal Sourcegraph teammates, join us in [`#cloud`](https://sourcegraph.slack.com/archives/C03JR7S7KRP) slack channel to ask questions or request help from our team.
```

Comment:
our handbook will automatically linkify text like #cloud to a link to the slack channel